crates/iota-sdk/README.md

```diff
@@ -145,9 +145,9 @@ See the programmable transactions [example](https://github.com/iotaledger/iota/b
    1. [Connect to IOTA Devnet](https://docs.iota.org/developer/getting-started/connect).
    1. [Make sure you have two addresses with gas](https://docs.iota.org/developer/getting-started/get-address) by using the `new-address` command to create new addresses:
       ```shell
-      iota client new-address ed25519
+      iota client new-address --key-scheme ed25519
```

Comment:
Do you want to keep the --key-scheme here so they have an example? Otherwise it could be removed

Reply:
I think I would keep it because there is the sentence about the different key schemes